back-end/seed_all_data.py

```
@@ -4,204 +4,268 @@
+                        'courses_id': row['courses_id'],
+                        'tutor_id': row['tutor_id'],
+                        'tutee_id': row['tutee_id'],
+                        'created_at': row['created_at'],
+                    }
+                )
+            except IntegrityError:
+                # FK 문제 또는 unique_together 충돌 시 발생
+                print_warning(
+                    f"⚠ 채팅방 ID {row['id']} 생성 중 무결성 오류 발생. 건너뜁니다."
+                )
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)